ci/benchmark/build.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# Copyright (c) 2018, NVIDIA CORPORATION.
+# Copyright (c) 2018-2020, NVIDIA CORPORATION.
```

Comment:
So, where are we adding a full (Apache version 2.0) copyright statement and where are we adding a one liner? Any criterion?

Reply:
Good question. The one-liners were auto-updated using the new script (`ci/checks/copyright.py`), and I added the missing ones with the full notice.  Does anyone have any guidance on which should be used where, and if so I can update accordingly.